scripts/troubleshoot/troubleshooterrors.sh

```diff
@@ -31,6 +31,7 @@ kubectlInstallLinkMessage="Please install kubectl as per the instructions https:
 jqInstallLinkMessage="Please install jq as per instructions https://stedolan.github.io/jq/download/ and rerun the troubleshooting script"
 ciExtensionReOnboarding="Please reinstall extension as per instructions https://docs.microsoft.com/en-us/azure/azure-monitor/containers/container-insights-enable-arc-enabled-clusters?toc=/azure/azure-arc/kubernetes/toc.json"
 timesyncHelpMessage="Please check if you have any timesync issues on your cluster nodes"
+dataCollectionRulesMessage="Please check this doc to setup dcr and dcra properly https://learn.microsoft.com/en-us/azure/azure-monitor/essentials/data-collection-rule-overview"
 
```

Comment:
we use powershell script not the sh file, so update the powershell script.

Reply:
added